news/8417.removal

```diff
@@ -0,0 +1 @@
+Deprecate support for Python 3.5
```

Comment:
Looks like you forgot to save the wording change ? Or am I missing something ?

Reply:
Oops, copy/paste error, will fix up later, thanks!